samples/dashboards/demos/climate/demo.js

```diff
@@ -160,6 +160,12 @@ async function setupBoard() {
         components: [{
             cell: 'time-range-selector',
             type: 'Highcharts',
+            // connector: {
```

Comment:
Is that a placeholder for the future or can be removed?

Reply:
It is not working, so I guess it needs to be removed. :/